.github/ISSUE_TEMPLATE/1. bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug 🐛
+about: '"I''ve seen it. It''s rubbish."'
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description
+<!-- A clear description of the bug -->
+
+
+
+
+## Expected Behavior
+<!-- What did you expect to happen instead? -->
+
+
+
+
+## Reproduction
+<!-- A minimal example that exhibits the behavior. -->
+
+
+
+
+## Environment
+<!-- Any additional information about your environment
+
+*Optionally run `prefect diagnostics` from the command line and paste the information here. -->
```

Comment:
```suggestion
* Optionally run `prefect diagnostics` from the command line and paste the information here. * -->
```

Reply:
Thanks - actually meant to delete all `*` because I don't think they render inside the comment!